modules/update-preparing-list.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-list_{context}"]
+= Removed Kubernetes APIs
+
+{product-title} 4.12 uses Kubernetes 1.25, which removed the following deprecated `v1beta1` and `v2beta1` APIs. You must migrate manifests and API clients to use the appropriate `v1` or `v2` API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Kubernetes documentation].
+
+.`v1beta1` and `v2beta1` APIs removed from Kubernetes 1.25
+[cols="2,2,1",options="header",]
+|===
+|Resource |API |Notable changes
```

Comment:
It might be helpful to add a "Migrate To" column with the replacement API for a quick preview of what needs to be done.

|Resource | API Version | Migrate To | Notable Changes |
| --- | --- | --- | --- |
| `CronJob` | `batch/v1beta1` | `batch/v1` | No |
| `EndpointSlice` | `discovery.k8s.io/v1beta1` | `discovery.k8s.io/v1` | Yes | 
| etc... |

Reply:
@sanchezl I could definitely do this, but what would you recommend for the `PodSecurityPolicy` entry, which doesn't have a simple version to migrate to?

Reply:
I would just write "Pod Security Admission", linked to https://kubernetes.io/docs/concepts/security/pod-security-admission/.  Notable changes should say "Yes" and link https://kubernetes.io/docs/reference/using-api/deprecation-guide/#psp-v125.